openstates/cli/update.py

```diff
@@ -364,6 +391,10 @@ def _tb_info(type, value, tb):  # type: ignore
         sys.excepthook = _tb_info
 
     logging.info(f"Module: {args.module}")
+
+    # delete all objects from S3 folder for current module
+    delete_all_objects_from_s3_folder(args.module)
```

Comment:
Do we have documentation somewhere around the logic of this flow? This pattern seems like a potential race condition to me, so having some clear explanation would be nice.

Reply:
The idea is to delete all files scraped from the former run before a new run starts for that jurisdiction. The base thought here is: only one instance of a jurisdiction runs in prod, that's why we clear the folder when the run starts. 

We also are not clearing this at the end of imports because we want to keep the data in case the runs failed or we need to do some form of validation after runs.

There's currently no documentation for this flow (or the realtime flag at large) - it's what @bfossen-ce and I agreed on in terms of the persistence for this data. We documentation as part todos realtime-flag

Reply:
got it, I'd add a comment for this now